magento2-nginx/fpm-7.0/etc/confd/templates/nginx/site_base.conf.tmpl

```diff
@@ -76,8 +76,9 @@ location /pub/ {
 }
 
 location /static/ {
-    # Uncomment the following line in production mode
-    # expires max;
+    {{ if eq "production" (getenv "MAGENTO_MODE") }}
+    expires max;
+    {{ end }}
```

Comment:
should it be clearer by reversing it to a not equal to dev? Otherwise it sounds like it only is applied to the production environment